app/components/RecipeModule/index.tsx

```diff
@@ -11,7 +11,7 @@ export default function RecipeModule() {
       return targetRecipe;
     }
   });
-  const ingredients = recipe && recipe.ingredients;
+  const ingredients = recie && recipe.ingredients;
```

Comment:
```suggestion
  const ingredients = recipe && recipe.ingredients;
```
I think you meant to type recipe.